var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -229,17 +235,13 @@ def install(self, spec, prefix):
         # install fortran
         self.install_component(find("fortran-installer", "*")[0])
 
-        # Some installers have a bug and do not return an error code when failing
-        if not is_exe(self.component_prefix.linux.bin.intel64.ifort):
-            raise RuntimeError("install failed")
-
     @run_after("install")
     def inject_rpaths(self):
         # Sets rpath so the compilers can work without setting LD_LIBRARY_PATH.
         patchelf = which("patchelf")
         patchelf.add_default_arg("--set-rpath", ":".join(self._ld_library_path()))
         for pd in ["bin", "lib", join_path("compiler", "lib", "intel64_lin")]:
-            for file in find(self.component_prefix.linux.join(pd), "*", recursive=False):
+            for file in find(self.component_prefix_linux.join(pd), "*", recursive=False):
```

Comment:
I believe inject_rpaths is a workaround for a problem that was fixed so I am bypassing the entire function for 2024: